exonum-java-binding/core/rust/src/utils/resource_manager/imp.rs

```diff
@@ -12,14 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use chashmap::CHashMap;
 use std::any::TypeId;
-use std::collections::HashMap;
-use std::sync::RwLock;
 
 use utils::Handle;
 
+const MAP_CAPACITY: usize = 512;
```

Comment:
I've chosen this value with the idea that reasonably big value will keep the load factor small enough spending not much memory to the initial pre-allocation.

Reply:
Wait, how the map can determine the load factor from the capacity (for an empty map)? These are supposedly different things. Why don't we either pass a load factor (if you have a really good intuition of which is right) or don't pass anything?

Reply:
Sorry for confusing you, I just meant that `CHashMap::with_capacity(N)` pre-allocates some amount of buckets depending on N, thus I supposed that defining some reasonable value here could prevent it from expensive re-allocations in future. In other words, it's rather a question if it is possible to estimate this or it is fully unpredictable and there is no sense in such optimizations.